.github/actions/templates/avm-setEnvironment/action.yml

```diff
@@ -44,8 +44,9 @@ runs:
         . (Join-Path $env:GITHUB_WORKSPACE 'utilities' 'pipelines' 'sharedScripts' 'Set-EnvironmentOnAgent.ps1')
 
         # Define PS modules to install on the runner
+        # Az.Accounts version 4.0.2 is specified to avoid the issue with the latest version the runners are using. This pin down can be removed once the agents are using this version by default.
```

Comment:
@sebassem @AlexanderSehr there's a PR open #4304 to add this module to the list of modules to be deployed using OIDC, now that the prerequisites for it have been fixed upstream. 

As this change here would trigger all pipelines, if required, would you agree to add this comment to #4304 and remove it from here so that this PR just triggers the sub-vending module once merged?

Reply:
Sounds like a good plan.

Reply:
The version pindown is not needed anymore, I reverted this change as the runners have been updated now to the latest module version